run_project_tests.py

```diff
@@ -503,6 +503,9 @@ def _run_tests(all_tests, log_name_base, extra_args):
         print('Could not determine number of CPUs due to the following reason:' + str(e))
         print('Defaulting to using only one process')
         num_workers = 1
+    # Due to Ninja deficiency, almost 50% of build time
```

Comment:
Can you clarify what kind of deficiency is being referred to?

Reply:
Basically it's #2082, but I'll add a better description if we actually consider this for merging (as opposed to Ninja fixing the actual issue).